File: mixedProblems/problem_2.py
# Problem 2: How do you calculate the number of vowels and consonants in a String

myString = input("Enter your desier String: ")
text = myString.lower()
noOfVowel = 0
noOfCons = 0

for word in text:
    if(word == 'a' or word == 'e' or word == 'i' or word == 'o' or word == 'u'):
        noOfVowel+=1
    else:
        noOfCons+=1

print(f"Number of Vowel is {noOfVowel} and Number of Consonant is {noOfCons}")

File: mixedProblems/problem_2.js
// Problem 2: How do you calculate the number of vowels and consonants in a String

myString = "Auntor"
noOfVowel = 0
noOfCons = 0
var arrayOfString = myString.toLowerCase().split("")
arrayOfString.forEach(element=>{
    if(element=='a' || element=='e' || element=='i'|| element=='o' || element=='u'){
        noOfVowel++
    }else{
        noOfCons++
    }
})

console.log(`No of Vowel is ${noOfVowel} and No of Consonent is ${noOfCons}`)

